index.html

```diff
@@ -797,6 +797,58 @@ <h2 id="BaseAudioContext">
             </p>
           </dd>
           <dt>
+            readonly attribute DOMHighResTimeStamp currentPerformanceTime
+          </dt>
+          <dd>
+            <p>
+              The <code>DOMHighResTimeStamp</code> representation of <a href=
+              "#widl-BaseAudioContext-currentTime"><code>currentTime</code></a>.
+            </p>
+            <p>
+              The <a href=
+              "#widl-BaseAudioContext-currentPerformanceTime"><code>currentPerformanceTime</code></a>
+              property contains the <code>DOMHighResTimeStamp</code>
+              representing the time in milliseconds from the <code>time
+              origin</code> (described in [[!hr-time-2]]) to the moment when
+              the <a href=
+              "#widl-BaseAudioContext-currentTime"><code>currentTime</code></a>
+              was last time updated and incremented internally by the duration
+              in milliseconds of the audio block processed by the system, so
+              that <code>currentTime</code> and
+              <code>currentPerformanceTime</code> values always correspond to
+              each other. If the context's rendering graph has not yet
+              processed a block of audio (i.e. <code>currentTime</code> has
+              never been updated), then <a href=
+              "#widl-BaseAudioContext-currentPerformanceTime"><code>currentPerformanceTime</code></a>
+              has a value of zero.
+            </p>
+            <p>
+              For blocks of 128 sample-frames audio block duration is equal to
+              roughly 3ms at a sample-rate of 44.1KHz, however the used audio
+              block duration depends on the system: on repeated calls of the
+              Web Audio API the reference is always the first call in a row.
+              For example if the system processes audio blocks of 512
+              sample-frames (calling the Web Audio API 4 times in a row) the
+              <code>currentPerformanceTime</code> value will be shifted by:
+            </p>
+            <pre>duration = 1000 * 4 * 128 / 44100Hz = 11.6ms</pre>
+            <p>
+              In order to keep the correspondence between
+              <code>currentTime</code> and <code>currentPerformanceTime</code>
+              attributes values the implementation should adjust the value of
+              the <code>currentPerformanceTime</code> shift according to the
+              current system block duration, if it varies over time.
```

Comment:
I don't understand what is being said here in "adjust the value of the... shift... if it varies over time". Why do we not simply define the way that `currentPerformanceTime` behaves in terms of the increments that are applied during the processing of each block? The behavior that I _think_ you are describing is just a natural consequence of the UA happening to process a bunch of blocks "in a row", but there is no need to describe any special behavior that the implementation should take like "adjusting a shift" -- if the UA processes 4 blocks (or any number of blocks) of 128 frames, then it should behave as though `currentTime` and `currentPerformanceTime` were incremented separately for each block.